### PR TITLE
Move RestartApiserver() into e2e/network

### PR DESCRIPTION
**Pull request by oomichi**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

The function is called from e2e/network test only, so this moves the function into the test for reducing e2e/framework/util.go code and removing invalid dependency on e2e test framework.

Ref: https://github.com/kubernetes/kubernetes/issues/77095
Ref: https://github.com/kubernetes/kubernetes/issues/81245

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/network/service.go [hunk 1]
```diff
@@ -3431,3 +3434,83 @@ func validateEndpointsPorts(c clientset.Interface, namespace, serviceName string
 	}
 	return fmt.Errorf("Timed out waiting for service %s in namespace %s to expose endpoints %v (%v elapsed)", serviceName, namespace, expectedEndpoints, framework.ServiceStartTimeout)
 }
+
+// restartApiserver restarts the kube-apiserver.
+func restartApiserver(namespace string, cs clientset.Interface) error {
+	// TODO: Make it work for all providers.
```
Comment (gavinfish) [inline test/e2e/network/service.go]: If we make it private here, is the TODO still valid?

File: test/e2e/framework/nodes_util.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func MasterUpgrade(f *Framework, v string) error {
 	case "gce":
 		return masterUpgradeGCE(v, false)
 	case "gke":
-		return masterUpgradeGKE(f.Namespace.Name, v)
+		return MasterUpgradeGKE(f.Namespace.Name, v)
```
Comment (tanjunchen) [inline test/e2e/framework/nodes_util.go]: Exposed?

File: test/e2e/network/service.go (hunk 1 above)
Comment (tanjunchen) [reply test/e2e/network/service.go]: /cc @wongma7 
for this TODO

Comment (oomichi) [reply test/e2e/network/service.go]: The test itself doesn't seem cloud provider specific, so I don't think we should remove this TODO at this time.

File: test/e2e/framework/nodes_util.go (hunk 2 above)
Comment (oomichi) [reply test/e2e/framework/nodes_util.go]: Yes, that is necessary to do this.

**Pull request merged**